app/helpers/encrypt.js

```diff
@@ -0,0 +1,8 @@
+const bcrypt = require('bcrypt');
+
+module.exports = data => {
+  const saltRounds = 5;
```

Comment:
Sorry I missed this, you shouldn't have hard code, this should be an environment variable.